doc/orchestrator.rst

```diff
@@ -165,3 +165,460 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multi-Orchestrator Example
+==========================
+SmartSim offers functionality to automate the deployment of multiple
+databases on an HPC cluster, supporting workloads that require multiple
+``Orchestrators`` for a ``Experiment``. For instance, a workload may consist of a
+simulation with high inference performance demands (necessitating a co-located deployment),
+along with an analysis and
+visualization workflow connected to the simulation (requiring a standard orchestrator).
+
+Below is a simple example demonstrating the process of:
+
+1. Launch two standard Orchestrators within the experiment with unique identifiers
+2. Launch one colocated Orchestrator within the experiment with a unique identifier
+3. Connect SmartRedis clients within the driver script to send tensors to std Orchestrators
+4. Connect SmartRedis clients within the application to retrieve tensors from std databases to store in colo databases
+
+The Application Script
```

Comment:
maybe do not separate the contents of the application script from the description?

Reply:
hmmm hmmm the description is for the Experiment in total! And the Experiment is comprised of two scripts, the experiment_script.py and application_script.py -> I think its easier if I indicate where I begin to talk about the application, but I think the "below" in the sentence starting the list is misleading, imma change it, and maybe add an overview section to create more separation

Reply:
I agree with @juliaputko that you should provide a very high level description of the two scripts before diving into the specific contents. I like the more detailed, functional summary, though as well.